scripts/src/public/auth.ts

```diff
@@ -11,7 +11,7 @@ async function getTokenAndUser(req: express.Request): Promise<[db.AuthToken, db.
 	}
 
 	const token = await db.AuthToken.findOneById(req.token);
-	if (!token) {
+	if (!token || !token.valid) {
```

Comment:
great catch. I wonder if clients will start failing or that the refresh mechanism will work @YohayBar ?

Reply:
Maybe we should also check the expiration here. I thought `valid` checked that too